### DISCUSSION TITLE
draw.io configure the desktop client only use Microsoft OneDrive as storage

### DISCUSSION THREAD

**Discussion by jakobiGerhard** [jgraph/drawio#4561] [Q&A]
Hello everyone, we would like to use the draw.io desktop app in our company. The software distribution will be done via SCCM. We are already using the plugin for Confluence and Jira. However, our security department is not satisfied with the possibility of integrating Google Drive, Google Docs, etc. as storage options. Is it possible to configure the client so that only Microsoft OneDrive can be used as a storage location? And then package this setup in SCCM for distribution?

**Comment by davidjgraph** [comment]
The desktop app doesn't connect to anything. You can only save locally.

**Reply by jakobiGerhard** [reply]
Thank you for the quick response. But it is possible to install plugins? And embed Google Docs, Slides, etc.?

**Reply by davidjgraph** [reply]
No, the desktop has a CSP applied that prevents all communications.

**Reply by jakobiGerhard** [reply]
OK, great. Is there any documentation available on this, or something we can provide to IT Security?
So, does this 'embedded' (see screenshot) only refer to the ability to embed the online version of draw.io in Google Drive and not the other way around, i.e., loading Google content into the desktop app?
![pic_draw io](https://github.com/user-attachments/assets/2942f6db-1181-44fd-830d-da0c1aaa0f30)

**Reply by davidjgraph** [reply]
1. Read the README - https://github.com/jgraph/drawio-desktop
2. Try linking those links.
